src/index.ts

```diff
@@ -1,4 +1,6 @@
 import { app, BrowserWindow } from 'electron';
+// new path for images starts here.....
+const path = require("path");
```

Comment:
*Suggestion*
Please remove this if this was part of the images issue and wasn't the solution.